src/components/Job/jobticket.scss

```diff
@@ -48,7 +48,10 @@
   &__right {
```

Comment:
you can use vs code extension Beautify css/sass/scss/less format file css

Reply:
i've changed